src/components/data/EntitiesContent.js

```diff
@@ -313,7 +314,7 @@ const EntitiesContent = ({
   };
 
   const downloadSelectedRows = async (columnSettings) => {
-    const tsv = buildTSV(columnSettings, selectedEntities);
+    const tsv = buildTSV(columnSettings, selectedEntities, true);
```

Comment:
I find raw boolean args hard to read without going to the method implementation.  There might be a more idiomatic Typescriptey approach than this suggestion, but a possible readability improvement would be to include an inline comment like this wherever you're using it:

```
buildTSV(columnSettings, selectedEntities, /* forDownload=*/ true)
```

Reply:
A common pattern in JS/TS is an "options object", which effectively provides named arguments.
https://www.codereadability.com/what-are-javascript-options-objects